src/main/java/org/templegalaxia/model/PointLoader.java

```diff
@@ -0,0 +1,72 @@
+package org.templegalaxia.model;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+public class PointLoader {
+  private List<float[]> pointArray = new ArrayList<>();
+  private int numPixels = 0;
+  private String resourceName;
+  private boolean loadPoints = true;
+
+  // NOTE(G3): There is a coordinate systems mismatched between rhino & Processing
+  private static String DEFAULT_COORDINATE_ORDER = "yzx";
+
+  public PointLoader(String resourceName) {
+    this(resourceName, DEFAULT_COORDINATE_ORDER);
+  }
+
+  public PointLoader(String resourceName, String coordinateOrder) {
+    DEFAULT_COORDINATE_ORDER = coordinateOrder;
```

Comment:
So the convention is that ALL_CAPS denotes a constant that won't get reassigned. The notion i had in mind was that you'd have like:

```
private static final String DEFAULT_COORDINATE_ORDER = "yxz";
private String coordinateOrder;
```

And then in the constructor you'd set `this.coordinateOrder = DEFAULT_COORDINATE_ORDER`  or `this.coordinateOrder = coordinateOrder`